packages/info/mixins/log/mixin.core.js

```diff
@@ -39,11 +40,11 @@ class LogMixin extends Mixin {
   }
   handleArguments(argv) {
     this.options = { ...this.options, ...argv };
-    const { quiet, verbose, _: commands } = this.options;
+    const { quiet = 0, verbose = 0, _: commands = [] } = this.options;
```

Comment:
We could also move this up to the constructor.

Reply:
Can you explain a bit more what you mean by that?

Reply:
It would not necessarily make things more readable, but it would probably make that additional `handleArguments`-call obsolete (at least if we also added similar code to all other mixins that are interested in `options`).

Reply:
I'm sorry, I don't understand. Can you maybe write a small example of what you would want to achieve?
As far as I can tell, we need the `handleArguments` call in the constructor, because it is responsible for setting the `logLevel` and we want to be able to set the logLevel in two cases - executing `un ..` should set the log level, that's why we have the mixin hook, that will be called through yargs and we wan to be able to set the log level through `initialize(config, options)` (or `configure(config, options)` on some public API), that's why we need to call the `handleArguments` method in the constructor.

Reply:
I agree, you are right here!